### ISSUE TITLE
CHANGE-ME link appears in system settings

### ISSUE THREAD

**Issue by javmarina** [firebase/FirebaseUI-Android#1389]
### Step 2: Describe your environment

  * Android device: Nokia 7 plus
  * Android OS version: 8.1.0
  * Google Play Services version: 15.0.1 - 16.0.0
  * Firebase/Play Services SDK version: 16.0.0
  * FirebaseUI version: 4.1.0
  
#### Steps to reproduce:

  1. Include FirebaseUI (auth) in a project
  2. Build and install
  3. Go to Settings > Apps > installed testing app > Open by default
  
#### Observed Results:

  * What happened?
In the "Supported links" line, you can see CHANGE-ME as a link. This is because Firebase UI includes this in the manifest. CHANGE-ME  is used for Github sign in, I think.
  
#### Expected Results:

  * What did you expect to happen?
I didn't expect this to appear, as it is a string related to a part of the library that I don't use. Moreover, users can see it too.
  
![screenshot_20180709-015538](https://user-images.githubusercontent.com/2737599/42425300-97d892ce-831b-11e8-8660-6affebc08dd7.png)

**Comment by SUPERCILEX** [comment]
Ouch, that was a miscalculation on my part, sorry! 😊 For a more permanent solution, I think we might want to make people opt-in to the GitHub stuff by enabling the manifest entry, but for now, this should fix it:
```xml
<activity
    android:name=".ui.provider.GitHubLoginActivity"
    tools:node="remove" />
```

**Comment by javmarina** [comment]
This is enough for now, thanks!

**Comment by noelmansour** [comment]
I'm not sure if this affects priority at all, but this bug breaks app links (like those used for passwordless sign up).

Also, I had to put the fully qualified activity name for the workaround to work:

```
<activity
    android:name="com.firebase.ui.auth.ui.provider.GitHubLoginActivity"
    tools:node="remove" />
```

**Comment by samtstern** [comment]
This was fixed and released in version `4.2.0`

**Comment by javmarina** [comment]
Thanks!
